Lib/test/test_call.py

```diff
@@ -33,9 +34,6 @@ def test_varargs2_ext(self):
         else:
             raise RuntimeError
 
-    def test_varargs0_kw(self):
-        self.assertRaises(TypeError, {}.__contains__, x=2)
-
```

Comment:
You may keep this test for other Python implementations. (This test doesn't check the error message.)